libs/events/src/types/toastMessages.ts

```diff
@@ -12,7 +12,7 @@ export interface ToastMessagePayloads {
   }
 
   [ToastMessageType.SWAP_POSTED_API]: {
-    orderUid: string
```

Comment:
I would keep `orderUid` to be consistent with other cowprotocol apps (backenders use `orderUid`)